src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro.mdx

```diff
@@ -16,64 +16,84 @@ import moreintegrationsNativeOtlpNoCollector from 'images/more-integrations_diag
+[instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/)
+for popular libraries and frameworks.
+
+To help you get started, we offer a simple example application in a variety of
+languages. For each language, the application is built using web framework popular
+to that language. The examples demonstrate how to configure:
+
+1. The service name which will display in New Relic.
+2. The OpenTelemetry Protocol (OTLP) exporter to send data to the
+   [New Relic OTLP endpoint](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp).
+3. Additional recommended settings to optimize OpenTelemetry for the best New
+   Relic experience.
+
+<TechTileGrid>
+  <TechTile
+    name="Go agent"
+    icon="logo-go"
+    to="https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/getting-started-guides/go"
+  />
 
-* **Directly from your app.** Each OpenTelemetry language SDK comes packaged with OTLP exporters. Configure the SDK to export via OTLP. Consult relevant OpenTelemetry documentation and ensure [configuration matches New Relic OTLP requirements](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/).
+  <TechTile
+    name="Java"
+    icon="logo-java"
+    to="https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/getting-started-guides/java"
+  />
 
-  <img
-  title="Diagram showing a direct export to New Relic to your app"
-  alt="Diagram showing a direct export to New Relic to your app"
-  src={moreintegrationsNativeOtlpNoCollector}
+  <TechTile
+    name=".NET"
+    icon="logo-dotnet"
+    to="https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/getting-started-guides/dotnet"
   />
 
-  <figcaption>
-    The OTLP exporter in your app or service can export directly to the New Relic OTLP endpoint.
-  </figcaption>
-* **Export from an OpenTelemetry Collector.** Configure the app to export to an intermediate OpenTelemetry collector, and from the collector to New Relic via OTLP. Ensure the collector's OTLP exporter [configuration matches New Relic OTLP requirements](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/). See [Collector for Processing](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-pipeline/opentelemetry-collector-pipeline-intro) for more details.
+  <TechTile
+    name="Node.js"
+    icon="logo-nodejs"
+    to="https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/getting-started-guides/javascript"
+  />
 
-  <img
-  title="Diagram showing the export to New Relic from a collector"
-  alt="Diagram showing the export to New Relic from a collector"
-  src={moreintegrationsNativeOtlpWithCollector}
+  <TechTile
+    name="Python"
+    icon="logo-python"
+    to="https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/getting-started-guides/python"
```

Comment:
Reminder to merge https://github.com/newrelic/newrelic-opentelemetry-examples/pull/598